website/docs/_advanced_install.mdx

```diff
@@ -363,6 +363,17 @@ Script to automatically download and cache standalone `scala-cli` launcher.
 </Tabs>
 </div></div>
 
+<SectionAbout title="Bootstrapped standalone fat JAR" colBigTitle="7"></SectionAbout>
+
+<div className="row"><div className="col col--9 col--offset-1 padding--lg advanced_install_methods">
+
+To run the bootstrapped Scala CLI standalone fat jar using Coursier, follow the command below:
```

Comment:
It may be worth it to explain briefly who may need this JAR and for what purpose (as I understand, it's mostly for corporate and/or no-internet-access uses).